rust/parser/test/mod.rs

```diff
@@ -1006,6 +1007,25 @@ fn test_parsing_pattern() {
     assert_valid_eq_repr!(expected, parsed, pattern);
 }
 
+#[test]
+fn test_parsing_patterns() {
```

Comment:
When do you use `_parsing_` or `_parse_`?

Reply:
I'd go with `parsing` everywhere.

Reply:
Renamed